google-cloud-notebooks/src/test/java/com/google/cloud/notebooks/v1beta1/it/ITNotebookServiceClientTest.java

```diff
@@ -0,0 +1,234 @@
+  private static final String LOCATION = "us-central1-a";
+  private static final String LOCATION_B = "us-central1-b";
+  private static final String PARENT = "projects/" + PROJECT_ID + "/locations/" + LOCATION;
+  private static NotebookServiceClient client;
+  private static final String ID = UUID.randomUUID().toString().substring(0, 8);
+  private static final String NOTEBOOK_INSTANCE_ID = "test-notebook-instance-id-" + ID;
+  private static final String ENVIRONMENT_ID = "test-environment-id-" + ID;
+  private static final String INSTANCE_NAME = PARENT + "/instances/" + NOTEBOOK_INSTANCE_ID;
+  private static final String ENVIRONMENT_NAME = PARENT + "/environments/" + ENVIRONMENT_ID;
+  private static Instance expectedNotebookInstance;
+  private static Environment expectedEnvironmentResponse;
+  private static final String MACHINE_TYPE_A = "n1-standard-4";
+  private static final String MACHINE_TYPE_B = "n1-standard-2";
+
+  @BeforeClass
+  public static void setUp() throws IOException, ExecutionException, InterruptedException {
+    // Create Test Notebook Instance
+    client = NotebookServiceClient.create();
+    ContainerImage containerImage =
+        ContainerImage.newBuilder().setRepository(FieldBehavior.OPTIONAL.name()).build();
+
+    Environment environment =
+        Environment.newBuilder()
+            .setName(ENVIRONMENT_NAME)
+            .setContainerImage(containerImage)
+            .build();
+
+    CreateEnvironmentRequest environmentRequest =
+        CreateEnvironmentRequest.newBuilder()
+            .setParent(PARENT)
+            .setEnvironmentId(ENVIRONMENT_ID)
+            .setEnvironment(environment)
+            .build();
+
+    expectedEnvironmentResponse = client.createEnvironmentAsync(environmentRequest).get();
+
+    Instance notebookInstance =
+        Instance.newBuilder()
+            .setContainerImage(containerImage)
+            .setMachineType(MACHINE_TYPE_A)
+            .build();
+    CreateInstanceRequest instanceRequest =
+        CreateInstanceRequest.newBuilder()
+            .setParent(PARENT)
+            .setInstanceId(NOTEBOOK_INSTANCE_ID)
+            .setInstance(notebookInstance)
+            .build();
+    expectedNotebookInstance = client.createInstanceAsync(instanceRequest).get();
+  }
+
+  @AfterClass
+  public static void tearDown() throws ExecutionException, InterruptedException {
+    // Delete Test Environment Instance
+    DeleteEnvironmentRequest deleteEnvironmentRequest =
+        DeleteEnvironmentRequest.newBuilder().setName(ENVIRONMENT_NAME).build();
+    client.deleteEnvironmentAsync(deleteEnvironmentRequest).get();
+    // Delete Test Notebook Instance
+    DeleteInstanceRequest deleteInstanceRequest =
+        DeleteInstanceRequest.newBuilder().setName(INSTANCE_NAME).build();
+    client.deleteInstanceAsync(deleteInstanceRequest).get();
+    client.close();
+  }
+
+  private String stopInstance() throws ExecutionException, InterruptedException {
```

Comment:
make it static

Reply:
These (startInstance/stopInstance) are private instance methods,  and I am using it with this object.